CHANGELOG/CHANGELOG-2.4.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 2.4
 
+## 2.4.1 - 2025-08-13
+
+### Added
+
+- Add resource requests for Windows container in Antrea deployment. ([#7254](https://github.com/antrea-io/antrea/pull/7254), [@XinShuYang])
+
+### Fixed
+
+- Add missing Run calls for nodeStore / serviceStore to avoid failed store lookups for FlowAggregator. ([#7343](https://github.com/antrea-io/antrea/pull/7343), [@antoninbas])
```

Comment:
This is not correct. Avoiding failed lookups is achieved by waiting for the stores to sync. And that's a secondary part of the patch. The key issue is that when `Run` is not called, garbage collection does not run on the stores and it's essentially a memory leak.

So a correct description would be:

> Add missing Run calls for nodeStore / serviceStore to start the garbage collection routines and fix a memory leak.